.github/workflows/test.yaml

```diff
@@ -231,10 +252,10 @@ jobs:
           rad group switch default
           rad env create default
           rad env switch default
-          rad recipe register default -e default -w default --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/rediscaches:latest --resource-type Applications.Datastores/redisCaches
-          rad recipe register default -e default -w default --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/mongodatabases:latest --resource-type Applications.Datastores/mongoDatabases
-          rad recipe register default -e default -w default --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/sqldatabases:latest --resource-type Applications.Datastores/sqlDatabases
-          rad recipe register default -e default -w default --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/rabbitmqqueues:latest --resource-type Applications.Messaging/rabbitMQQueues
+          rad recipe register default -e default -w default --template-kind bicep --template-path ghcr.io/radius-project/recipes/local-dev/rediscaches:${{ steps.gen-radius-version.outputs.RADIUS_VERSION }} --resource-type Applications.Datastores/redisCaches
```

Comment:
we were always using latest for these, we should use the release version or edge depending on the branch

Reply:
If we have a `workflow_dispatch` event, `RADIUS_VERSION` would be associated with that event. But we don't publish local-dev recipes with that tag correct? So this might fail on workflow dispatch events?

Reply:
`RADIUS_VERSION` will just be `edge` or `0.36` or whatever, the actual event that triggered it shouldn't matter. so it shouldn't fail on workflow_dispatch

Reply:
ah I see, bc its taking the input version not a dispatch specific version